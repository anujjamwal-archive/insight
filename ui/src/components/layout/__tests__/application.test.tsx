import * as React from "react";
import { ApplicationLayout } from "..";
import { shallow } from "enzyme";

describe(ApplicationLayout, () => {
  it("should render correctly", () => {
    const layout = shallow(<ApplicationLayout />);

    expect(layout).toMatchSnapshot();
  });
});
