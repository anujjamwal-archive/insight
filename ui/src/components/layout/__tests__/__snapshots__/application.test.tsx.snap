// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render correctly 1`] = `
<div>
  <AppBar>
    <Text>
      Insight
    </Text>
  </AppBar>
  <div />
</div>
`;
